assets/controller.yaml

```diff
@@ -118,6 +118,7 @@ spec:
           args:
             - --csi-address=$(ADDRESS)
             - --default-fstype=ext4
+            - --timeout=300s
```

Comment:
I can see upstream has the same timeout in resizer and snapshotter, IMO it's worth bumping them too.

Reply:
agreed.

Reply:
fixed